### PR TITLE
Add support for running Firestore integration tests

### PR DESCRIPTION
**Pull request by wilhuff**
Part of #434.

### CODE REVIEW

**Comment by wilhuff** [comment]
Fixed.

File: scripts/build.sh [hunk 1]
```diff
@@ -314,6 +314,12 @@ case "$product-$method-$platform" in
         "${xcb_flags[@]}" \
         build \
         test
+
+    RunXcodebuild \
+        -workspace 'Firestore/Example/Firestore.xcworkspace' \
+        -scheme "Firestore_IntegrationTests_$platform" \
+        "${xcb_flags[@]}" \
+        build
     ;;
```
Comment (paulb777) [inline scripts/build.sh]: Is adding test for a future PR?

Comment (wilhuff) [reply scripts/build.sh]: We've never had Firestore integration tests run on Travis. I haven't figured that out yet. When I do, I'll add to all the platforms we support.

Comment (paulb777) [reply scripts/build.sh]: See https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/install_prereqs.sh#L47

**Pull request merged**